{"name": "test_negative_login_with_Wrong_User_Information_1___aws_gmail_com____q__________________", "status": "broken", "statusDetails": {"message": "TypeError: TestLoginFunctionality.test_negative_login_with_Wrong_User_Information() takes 3 positional arguments but 7 were given", "trace": "self = <tests.test_loginTest.TestLoginFunctionality testMethod=test_negative_login_with_Wrong_User_Information_1___aws_gmail_com____q__________________>\n\n    @wraps(func)\n    def wrapper(self):\n>       return func(self, *args, **kwargs)\nE       TypeError: TestLoginFunctionality.test_negative_login_with_Wrong_User_Information() takes 3 positional arguments but 7 were given\n\nvenv\\Lib\\site-packages\\ddt.py:220: TypeError"}, "start": 1677149215250, "stop": 1677149215251, "uuid": "82f7e010-5ddf-4b43-b434-e7dd4ba73a8c", "historyId": "91b3a710abaf401ea8cbd3d252aece06", "testCaseId": "9930cae52293c04dbf423e50eb43b9b7", "fullName": "tests.test_loginTest.TestLoginFunctionality#test_negative_login_with_Wrong_User_Information_1___aws_gmail_com____q__________________", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('setup')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_loginTest"}, {"name": "subSuite", "value": "TestLoginFunctionality"}, {"name": "host", "value": "DESKTOP-75D171T"}, {"name": "thread", "value": "13816-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_loginTest"}]}